lib/internal/errors.js

```diff
@@ -115,9 +107,35 @@ const prepareStackTrace = (globalThis, error, trace) => {
     return errorString;
   }
   return `${errorString}\n    at ${ArrayPrototypeJoin(trace, '\n    at ')}`;
-};
+}
+
+function setInternalPrepareStackTrace(callback) {
+  internalPrepareStackTrace = callback;
+}
+
+/**
+ * Every realm has its own prepareStackTraceCallback. When `error.stack` is
+ * accessed, if the error is created in a shadow realm, the shadow realm's
+ * prepareStackTraceCallback is invoked. Otherwise, the principal realm's
+ * prepareStackTraceCallback is invoked. Note that accessing `error.stack`
+ * of error objects created in a VM Context will always invoke the
+ * prepareStackTraceCallback of the principal realm.
+ * @param {object} globalThis The global object of the realm that the error was
+ *   created in. When the error object is created in a VM Context, this is the
+ *   global object of that VM Context.
+ * @param {object} error The error object.
+ * @param {*} trace
```

Comment:
Is it really anything? Not `string` or `string[]`?

Reply:
Not actually, it is an array of V8 internal CallSite objects: https://v8.dev/docs/stack-trace-api#customizing-stack-traces.

I've updated the comment to link to the V8 doc.